doc/content/examples/example6a.md

```diff
@@ -0,0 +1,65 @@
+# Example 6a: Displacement Controlled Contact Problem with elastic soil
+
+## Model Description
+
+This example demonstrates the use of the contact block to simulate a frictional contact between a solid block and an elastic soil in MASTODON. A solid cubical block of dimensions 6" each side  is placed in contact with an cubical soil block of dimensions 48" each side. The remaining three sides of the soil are fixed and the solid block is displaced by a prescribed distance in the direction parallel to the contact surface. The normal forces and the frictional forces obtained from MASTODON is compared with the analytical results.
+
+
+## Modeling in MASTODON
+
+!listing examples/ex06a/psf_grav.i
+
+The mesh consisting of the solid block and the soil is generated by importing an existing mesh file [fig:ex06input].
+The details about the type of mesh file supported can be found at [GettingStarted](manuals/user/index.md). The displacements are defined in the variables section and the velocity, acceleration, normal forces and the tangential (frictional) forces are defined as the auxiliary variables.
```

Comment:
The mesh used in this model was generated externally using the mesh generation software, Cubit. MASTODON imports the mesh using the 'FileMesh' type specified in the 'Mesh' block. Details regarding supported mesh file types can be found here...

The displacement variables are defined in the 'Variables' block. The accelerations, velocities, normal and tangential (frictional) forces are defined as auxiliary variables in the 'AuxVariables' block.